src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/building/ToolBuildingRecipe.java

```diff
@@ -31,10 +31,15 @@
 import slimeknights.tconstruct.tables.TinkerTables;
 
 import javax.annotation.Nullable;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.IntStream;
 
+import static slimeknights.tconstruct.plugin.jei.ToolBuildingCategory.SLOT_SIZE;
+import static slimeknights.tconstruct.plugin.jei.ToolBuildingCategory.X_OFFSET;
+import static slimeknights.tconstruct.plugin.jei.ToolBuildingCategory.Y_OFFSET;
```

Comment:
You should not reference JEI code outside of JEI plugins. Move these constants to this class (or copy if they existed before your PR)